### DISCUSSION TITLE
ReferenceError "'Clear' is not defined" when using livewire.esm

### DISCUSSION THREAD

**Discussion by kellyguelce** [livewire/livewire#6218] [7. Bugs (archived)]
### Livewire version

3.0.0-beta.4

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1.Import Livewire and Alpine in app.js (Manually bundling Livewire and Alpine)
```
// /resources/js/app.js

import { Livewire, Alpine } from '../../vendor/livewire/livewire/dist/livewire.esm'

Livewire.start()
```
2. Create a livewire component
`php artisan make:livewire someComponent`

3. Make this component an alpinejs component
```
<div x-data='alpineComponent'>
    // content
</div>
```
4. Create a file for the alpine component : `/resources/js/components/AlpineComponent.js` 
```
export default () => ({
    // code
})
```

4. Registering the Alpine component
```
// /resources/js/app.js

import { Livewire, Alpine } from '../../vendor/livewire/livewire/dist/livewire.esm'
import alpineComponent from './components/AlpineComponent.js'

Alpine.data('alpineComponent', alpineComponent)

Livewire.start()
```
5. Now add a link to another page. Example : `<a href="{{ route('page2') }}" wire:navigate.hover>Page 2</a>`
Now the error should show up everytime the mouse hovers then leaves the link.
The error shows up only when i use `wire:navigate.hover`.

### Additional information

ERROR: livewire.esm.js?t=1691278293518:5185  Uncaught ReferenceError: clear is not defined
    at HTMLAnchorElement.handler (livewire.esm.js?t=1691278293518:5185:11)

Apparently, the error is in this file `/vendor/livewire/livewire/dist/livewire.esm` at line 5185
```
5180  function whenThisLinkIsHoveredFor(el, ms = 60, callback) {
5181     el.addEventListener("mouseenter", (e) => {
5182         let timeout = setTimeout(() => {
5183     }, ms);
5184     let handler = () => {
5185       clear; 
5186       el.removeEventListener("mouseleave", handler);
5187      };
5188     el.addEventListener("mouseleave", handler);
5189         callback(e);
5190     });
5191  }
```

What is `clear;` at line 5185 ?
Is it a syntax error for `window.clear` or `console.clear()` ?

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@kellygue I think this has already been fixed. Can you update to beta.7 and test again and let us know how it goes? Thanks!

**Reply by kellyguelce** [reply]
I still have the same error with the latest version. I simply commented out the line, and it works perfectly 😄 Thank you

**Reply by joshhanley** [reply]
@kellygue ok thanks! I will get Caleb to have a look.
